internal/testrunner/runners/system/servicedeployer/kubernetes.go

```diff
@@ -132,15 +132,17 @@ func (ksd KubernetesServiceDeployer) installCustomDefinitions() error {
 
 var _ ServiceDeployer = new(KubernetesServiceDeployer)
 
-func findKubernetesDefinitions(definitionsDir string) ([]string, error) {
-	files, err := filepath.Glob(filepath.Join(definitionsDir, "*.yaml"))
-	if err != nil {
-		return nil, errors.Wrapf(err, "can't read definitions directory (path: %s)", definitionsDir)
+func findKubernetesDefinitions(definitionsDir string) (string, error) {
+	if _, err := os.Stat(definitionsDir); err != nil {
+		return "", errors.Wrapf(err, "can't read definitions directory (path: %s)", definitionsDir)
 	}
-
-	var definitionPaths []string
-	definitionPaths = append(definitionPaths, files...)
-	return definitionPaths, nil
+	if _, err := os.Stat(filepath.Join(definitionsDir, "kustomization.yaml")); err != nil {
+		if _, err := os.Stat(filepath.Join(definitionsDir, ".empty")); err != nil {
+			return "", errors.Errorf("kustomization.yaml file is missing (path: %s)", definitionsDir)
```

Comment:
maybe we should also say the alternative way to fix this. Either add a .empty file or add a kustomization.yaml.